### PR TITLE
Refactor jobpost app, event app, add debug toolbar, add LOGIN_URL

### PR DESCRIPTION
**Pull request by hamblin**
Many changes, separated in 4 commits, rolled into one PR:

NOTE: this PR has backwards incompatible migration changes in 'jobpost' and 'event' apps (so DB should be recreated) and also requires installing django-debug-toolbar while DEBUG=True.  Due to these significant changes, I'm requesting review from @benjaoming before they are merged.

**Summary of changes:**

Refactor 'event' app
- renamed model field names
- related template changes
- delete old migrations and create new migrations for event app

Refactor 'jobpost' app
- renamed model field names
- moved ModelForm to forms.py
- updated templates to reflect new model field names (in this app and others)
- delete old migrations, create new migrations for jobpost app

Add django-debug-toolbar for help with dev/debugging:
- add to requirements_dev.txt
- changes to settings/dev.py required for debug toolbar
- modify djangodenmark/urls.py to add debug toolbar URLs when DEBUG = True

Add LOGIN_URL = 'login' (named URL) because default value ('/accounts/login/') does not work based on current url structure.  (this future-proofs things as well... as long as 'login' is still the named URL for desired login page)

Various changes to some template files just to restructure HTML/CSS for layout purposes.

### CODE REVIEW

File: jobpost/models.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
 from django.contrib.auth.models import User
-from django.core.validators import RegexValidator
 from django.db import models
-from django.forms import ModelForm
+from django.urls import reverse
 
 
 # Available titles for searching jobs
-JOBTYPE_CHOICES = (
+job_type_CHOICES = (
```
Comment (benjaoming) [inline jobpost/models.py]:
```suggestion
JOBTYPE_CHOICES = (
```

File: jobpost/models.py [hunk 2]
```diff
@@ -15,52 +14,27 @@
...
-    jobLocation = models.CharField(max_length=100)
-    jobContactPerson = models.CharField(max_length=80)
-    jobApplyHere = models.URLField(
-        max_length=100,
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    job_title = models.CharField(max_length=60, verbose_name="Job title")
+    job_company_name = models.CharField(max_length=100, verbose_name="Company name")
+    job_description = models.TextField(verbose_name="Describe the position")
+    job_type = models.CharField(
+        max_length=11, choices=job_type_CHOICES, verbose_name="Job type"
```
Comment (benjaoming) [inline jobpost/models.py]:
```suggestion
        max_length=11, choices=JOBTYPE_CHOICES, verbose_name="Job type"
```

**Review by benjaoming — APPROVED** [review]
:clap:

**Comment by hamblin** [comment]
Good catch on the JOBTYPE_CHOICES... I got a bit carried away with my find/replace.  

I'll merge the PR.

**Comment by benjaoming** [comment]
Great! The staging server is updated now :)

**Pull request merged**